ionos_dev/search-engine/search/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dome-search-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: dome-search
+      app.kubernetes.io/name: dome-search
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: dome-search
+        app.kubernetes.io/name: dome-search
+    spec:
+      containers:
+        - name: search
+          image: mardetom/search:0.0.2
```

Comment:
Where can I find the source to that image? And why is no chart used for it? With that, no additional configuration(f.e. log-level, metrics endpoints) can be configured

Reply:
The image is public in docker-hub (https://registry.hub.docker.com/r/mardetom/search/tags). Maybe we can also add "imagePullPolicy: IfNotPresent"

Reply:
I was searching for the source code. According to the rules in DOME it should be open source. I wanted to take a look at the configuration options and general documentation of the component.

Reply:
The source code is at this link https://github.com/DOME-Marketplace/search

Reply:
I updated the repo of the "search"; it's public in Harbor at the link production.eng.it:8433/dome/search:0.0.6. 
You can try via pull => docker pull production.eng.it:8433/dome/search:0.0.6